Miscellaneous Operations on Your Skywallet.md

```diff
@@ -0,0 +1,73 @@
+# Miscellaneous Operations on Your Skywallet
+
+In previous chapters, we saw how to initialize and personalize your Skywallet. However, you can also do some additional operations on your Skywallet like: 
```

Comment:
Use `perform` rather than `do` please. I think that sounds better. Also consider removing the first sentence. Not all users will run through the manual in one go and often they may come back to only a specific sentence. Then these intro sentences will not be beneficial.